Contributing.md

```diff
@@ -0,0 +1,37 @@
+# Contributing 
+
+### All conversations and interactions should be respectful and harassment free
+
+## Issues
+If something doesn't work or you cannot figure out how to make it work, open an ``Issue``.
+
+ See if the issue already exists, if it does either try to get help from the comments in its discussions or comment your specific needs. If it doesn't exist then open a new issue.
+
+Some good things to structure your issue
+
+1. Describe your problem as fully as possible
+2. Include screenshots of what result is being shown and what you expected 
+3. If possible include your local environment specifications 
+4. If you want to suggest a new feature that doesn't exist, describe your suggestion as much as you can and tag the creator or core members of the repo to have a discussion. 
+***
+
+## Contributing to Codebase
+If you want to develop/solve issues for spotivy follow these steps if you are new/beginner to our codebase
+
+1. `Fork` the repo on your personal github account
+2. `Clone` the repo in your local machine 
+3. Code and hack away for any issues or new features
+4. write descriptive `commit` messages 
+5. make sure to test your code before you send a PR
+6. `Push` to your remote branch
+7. Finally create a `pull request` to  ```develop``` branch
```

Comment:
Please use capital letters on every beginning.

Also, only one tick ("`") is necessary to format code on markdown.